app/views/comments/_comment.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if comment && comment.user %>
-  <% if comment.depth < 3 %>
+  <% if comment.depth < 3 && !(should_be_hidden?(comment, @root_comment) && comment.depth.zero?) %>
```

Comment:
I'm seeing top level comments here when they should be hidden altogether

<img width="885" alt="Screen Shot 2019-12-02 at 5 15 52 PM" src="https://user-images.githubusercontent.com/3102842/69999805-822b3900-1527-11ea-90e7-20b245dd8dbb.png">